http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/GrpcResource.java

```diff
@@ -20,6 +28,50 @@ public class GrpcResource {
     @GrpcClient("hello")
     Greeter client;
 
+    @Inject
+    @GrpcClient("reflection-service")
+    MutinyServerReflectionGrpc.MutinyServerReflectionStub reflection;
+
+    @GET
+    @Path("reflection/service/info")
```

Comment:
Both `@Path`s you added do not start with a slash. That's absolutely fine (though personally I'm used to leading slash), but we need to stick to one convention inside the file (or ideally module).

There is already endpoint `hello` inside this resource whose path starts with the slash. Please see https://github.com/quarkus-qe/quarkus-test-suite/blob/main/http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/GrpcResource.java#L24

I suggest `/reflection/service/info` and `reflection/descriptor/{profFilename}` (or change the `hello` path).

Thanks